__tests__/fetcher/api_mockdata.ts

```diff
@@ -0,0 +1,2851 @@
+export const articleUuidMock = {
+  __typename: 'Article',
+  id: 27801,
+  alias:
+    '/mathe/zahlen-groessen/bruchrechnen-dezimalzahlen/dezimalbrueche/addition-subtraktion-dezimalbruechen',
+  instance: 'de',
+  license: {
+    id: 1,
+    url: 'https://creativecommons.org/licenses/by-sa/4.0/deed.de',
+    title: 'Dieses Werk steht unter der freien Lizenz CC BY-SA 4.0',
+  },
```

Comment:
```suggestion
    '/math/numbers/rational-numbers',
  instance: 'en',
  license: {
    id: 1,
    url: 'https://creativecommons.org/licenses/by-sa/4.0',
    title: 'This content is licensed under CC BY-SA 4.0',
  },
```

English versions ware often a better example (in case a non German speaking programmer has to use this code in the future).